docs/index-shared2.asciidoc

```diff
@@ -0,0 +1,111 @@
+
+// Breaking Changes
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/breaking-changes.asciidoc
+include::static/breaking-changes.asciidoc[]
+
+// Upgrading Logstash
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/upgrading.asciidoc
+include::static/upgrading.asciidoc[]
+
+// Configuring Logstash
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/configuration.asciidoc
+include::static/configuration.asciidoc[]
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/multiple-pipelines.asciidoc
+include::static/multiple-pipelines.asciidoc[]
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/reloading-config.asciidoc
+include::static/reloading-config.asciidoc[]
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/managing-multiline-events.asciidoc
+include::static/managing-multiline-events.asciidoc[]
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/glob-support.asciidoc
+include::static/glob-support.asciidoc[]
+
+// Working with Filebeat Modules
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/filebeat-modules.asciidoc
+include::static/filebeat-modules.asciidoc[]
+
+// Transforming Data
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/transforming-data.asciidoc
+include::static/transforming-data.asciidoc[]
+
+// Deploying & Scaling
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/deploying.asciidoc
+include::static/deploying.asciidoc[]
+
+// Troubleshooting performance
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/performance-checklist.asciidoc
+include::static/performance-checklist.asciidoc[]
+
+// Monitoring overview
+
+:edit_url: https://github.com/elastic/logstash/edit/master/docs/static/monitoring.asciidoc
+include::static/monitoring.asciidoc[]
```

Comment:
This topic has a couple links (lines 27 and 41) that don't resolve correctly in your build because they use `{xpack}` instead of `{xpack-ref}`.